commune_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,116 @@
+# Generated by Django 4.1.3 on 2022-12-27 02:35
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
```

Comment:
This code already merged as part of:
https://github.com/redhat-beyond/commune/pull/27

please rebase

Reply:
And add tests